D1673/P1673.md

```diff
@@ -6764,82 +6534,59 @@ public:
     offset(data_handle_type p, size_t i) const
       noexcept(noexcept(@_nested-accessor_@.offset(p, i)));
 
-  constexpr Accessor nested_accessor() const;
+  constexpr const Accessor& nested_accessor() const;
 };
 ```
 
-[2]{.pnum} `Accessor` meets the `mdspan` accessor policy requirements
-**[mdspan.accessor.reqmts]**.
+[2]{.pnum} *Mandates:*
 
-```c++
-using reference = /* see below */;
-```
+  * [2.1]{.pnum} `is_copy_constructible_v<element_type>` is `true`.
 
-[3]{.pnum} This names
+  * [2.2]{.pnum} `Accessor` meets the accessor policy requirements **[mdspan.accessor.reqmts]**.
 
-* [3.1]{.pnum} `typename Accessor::reference`,
-    if `remove_cv_t<typename Accessor::element_type>` is an arithmetic type;
+  * [2.3]{.pnum} _`scalable`_`<ScalingFactor, typename Accessor::element_type>` is `true`. 
 
-* [3.2]{.pnum} otherwise,
-    _`conjugated-scalar`_`<typename Accessor::reference, remove_cv_t<typename Accessor::element_type>>`.
 
 ```c++
-using element_type = /* see below */
+constexpr conjugate_accessor(const Accessor& acc);
 ```
 
-[4]{.pnum} Let the exposition-only type `pre-const-element-type` name
-
-* [4.1]{.pnum} `typename Accessor::element_type`,
-    if `remove_cv<typename Accessor::element_type>` is an arithmetic type;
-
-* [4.2]{.pnum} otherwise, `reference::value_type`.
-
-[5]{.pnum} Then, `element_type` names `add_const_t<`_`pre-const-element-type`_`>`.
-
-<i>[Note:</i> `conjugate_accessor` provides read-only access. <i>-- end note]</i>
+[3]{.pnum} *Effects:* Direct non-list-initializes
+_`nested-accessor`_ with `acc`.
 
 ```c++
-using offset_policy = /* see below */;
+template<class OtherScalingFactor, class OtherNestedAccessor>
+  constexpr conjugate_accessor(const conjugate_accessor<OtherScalingFactor, OtherNestedAccessor>& other);
 ```
 
-[6]{.pnum} This names
-
-* [6.1]{.pnum} `typename Accessor::offset_policy`,
-    if `remove_cv_t<typename Accessor::element_type>` is an arithmetic type;
+[4]{.pnum} *Constraints:* `is_constructible_v<`_`nested-accessor`_`, const OtherNestedAccessor&>` is `true`, and
```

Comment:
I think we're missing the constraint for `OtherScalingFactor` here.  More importantly, _`nested-accessor`_ is a data member; we need to use a type there.

```suggestion
[4]{.pnum} *Constraints:*

  * [4.1]{.pnum} `is_constructible_v<Accessor, const OtherNestedAccessor&>` is `true`.

  * [4.2]{.pnum} `is_constructible_v<ScalingFactor, const OtherScalingFactor&>` is `true`.
```

This also suggests that it might be helpful to rename the template argument `Accessor` to `NestedAccessor`, so that the naming pattern is consistent.